tests/aws/test_s3.py

```diff
@@ -0,0 +1,36 @@
+import os
+import uuid
+from unittest import TestCase
+
+from moto import mock_s3
+
+from hip_data_tools.authenticate import AwsConnection
+from hip_data_tools.aws.s3 import S3Util
+
+
+class TestS3Util(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.sample_file_location = "./test_sample.txt"
+        cls.sample_file_content = str(uuid.uuid4())
+        with open(cls.sample_file_location, 'w+') as f:
+            f.write(cls.sample_file_content)
+
+    @classmethod
+    def tearDownClass(cls):
+        os.remove(cls.sample_file_location)
+
+    @mock_s3
+    def test_should__upload_then_download_file_from_s3__when_using_s3util(self):
```

Comment:
I think the word "test_" is redundant

Reply:
this is required by the unittest framework to discover tests